tensorflow/tensorflow.bzl

```diff
@@ -867,18 +871,29 @@ def tf_mkl_kernel_library(name,
                           deps=None,
                           alwayslink=1,
                           copts=tf_copts(),
+                          nocopts="-fno-exceptions",
                           **kwargs):
-  if_mkl(
-      tf_kernel_library(
-          name,
-          prefix=prefix,
+    if not bool(srcs):
+        srcs = []
+    if not bool(hdrs):
+        hdrs = []
+
+    if prefix:    
+        srcs = srcs + native.glob(
+            [prefix + "*.cc"])
+        hdrs = hdrs + native.glob(
+            [prefix + "*.h"])
+
+    if_mkl(
+      native.cc_library(
+          name=name,
           srcs=srcs,
-          gpu_srcs=gpu_srcs,
           hdrs=hdrs,
           deps=deps,
           alwayslink=alwayslink,
-          copts=copts,
-          **kwargs))
+          copts=tf_copts(),
```

Comment:
This should be copts=copts.
It is possible someone feeds different copts through the args, and we do not want them to be overridden.

Reply:
Done.